diff -Nur xerces-2_4_0.orig/src/org/apache/html/dom/HTMLFrameElementImpl.java xerces-2_4_0.chng/src/org/apache/html/dom/HTMLFrameElementImpl.java
--- xerces-2_4_0.orig/src/org/apache/html/dom/HTMLFrameElementImpl.java	Thu Mar 27 21:03:13 2003
+++ xerces-2_4_0.chng/src/org/apache/html/dom/HTMLFrameElementImpl.java	Fri Jun 27 11:09:05 2003
@@ -168,6 +168,10 @@
         setAttribute( "src", src );
     }
 
+    public Document getContentDocument()
+    {
+	throw new java.lang.UnsupportedOperationException();
+    }
     
     /**
      * Constructor requires owner document.
diff -Nur xerces-2_4_0.orig/src/org/apache/html/dom/HTMLIFrameElementImpl.java xerces-2_4_0.chng/src/org/apache/html/dom/HTMLIFrameElementImpl.java
--- xerces-2_4_0.orig/src/org/apache/html/dom/HTMLIFrameElementImpl.java	Thu Mar 27 21:03:12 2003
+++ xerces-2_4_0.chng/src/org/apache/html/dom/HTMLIFrameElementImpl.java	Fri Jun 27 11:09:42 2003
@@ -191,6 +191,10 @@
         setAttribute( "width", width );
     }
 
+    public Document getContentDocument()
+    {
+	throw new java.lang.UnsupportedOperationException();
+    }
     
     /**
      * Constructor requires owner document.
diff -Nur xerces-2_4_0.orig/src/org/apache/html/dom/HTMLObjectElementImpl.java xerces-2_4_0.chng/src/org/apache/html/dom/HTMLObjectElementImpl.java
--- xerces-2_4_0.orig/src/org/apache/html/dom/HTMLObjectElementImpl.java	Thu Mar 27 21:03:11 2003
+++ xerces-2_4_0.chng/src/org/apache/html/dom/HTMLObjectElementImpl.java	Fri Jun 27 11:10:29 2003
@@ -281,7 +281,10 @@
         setAttribute( "width", width );
     }
 
-    
+    public Document getContentDocument()
+    {
+	throw new java.lang.UnsupportedOperationException();
+    }
     
     /**
      * Constructor requires owner document.
